### ISSUE TITLE
Make an agora-specific log system

### ISSUE THREAD

**Issue by Geod24** [bosagora/agora#290]
Currently, Agora relies on `logInfo` / `logDebug` & co from Vibe.d for logging.
This is problematic, as our logging is interleaved with Vibe.d's network-level logging, which we rarely want to see. It's also impossible to categorize output by topic or by module, as Tango loggers allow to.
The solution to this issue is not necessarily home-brewed, as `std.experimental.logger` exists.

**Comment by AndrejMitrovic** [comment]
Also it would be great to hook SCP's logging to whatever logger we implement.

Currently it's:

```
#define CLOG(LEVEL, MOD) std::cout << "[" << LEVEL << ", " << MOD << "] "
```

This makes the mixed vibe.d/stdout logging look bad.

**Comment by Geod24** [comment]
Yup, that was a yuuuuge hack to reduce dependencies

**Comment by Geod24** [comment]
Started to work on this as it makes re-enabling system integration test much easier.

**Comment by Geod24** [comment]
Fixed by #314
